.github/workflows/gemini-pr-review.yml

```
@@ -1,456 +1,222 @@
-
-          # Extract additional instructions from comment
-          ADDITIONAL_INSTRUCTIONS="$(
-            echo "${COMMENT_BODY}" | sed 's/.*@gemini-cli \/review//' | xargs
-          )"
-          echo "additional_instructions=${ADDITIONAL_INSTRUCTIONS}" >> "${GITHUB_OUTPUT}"
+          GEMINI_API_KEY: ${{ secrets.GEMINI_API_KEY }}
+        run: |
+          set -o pipefail
+
+          ADV_PROMPT=$(cat <<'PROMPT'
+[role Senior Staff Engineer | Principal Reviewer]
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 119: could not find expected ':'

(syntax-check)